charts/redpanda/templates/_helpers.tpl

```diff
@@ -505,7 +537,7 @@ advertised-host returns a json sring with the data neded for configuring the adv
 */}}
 {{- define "warnings" -}}
   {{- $result := list -}}
-  {{- $warnings := list "redpanda-memory-warning" -}}
+  {{- $warnings := list "redpanda-memory-warning" "redpanda-cpu-warning" -}}
```

Comment:
thank you

Reply:
weird.. I do see it added in https://github.com/redpanda-data/helm-charts/blob/2bf7ceebcd3e827c87d5a3918346a8217b44efbb/charts/redpanda/templates/_helpers.tpl#L540

Reply:
going to mark it as resolved - let me know if you are still seeing missing "redpanda-cpu-warning" in the `$warnings ` list

Reply:
ah.. derp.. nevermind :)